requirements.txt

```diff
@@ -1,9 +1,9 @@
 # core dependencies
-pytorch==1.2.0
+pytorch==1.0.2
```

Comment:
Why downgrading ? Actually v1.3 is out which I will take care of update later this week

Reply:
1.0.2 was just the latest in PyPI. I see now, pytorch actually recommends installing from their archive with this:

```
pip3 install torch===1.3.0 torchvision===0.4.1 -f https://download.pytorch.org/whl/torch_stable.html
```

I'm pretty new to pytorch actually.

Reply:
@edgarriba I think that "release" version == "pip version" of kornia should be compatible with pip version of pytorch. Otherwise it is kind of weird.
And the "dev" version == master might be as high as we want :)

Reply:
Makes sense. Will setup it up like that